src/libraries/System.Private.CoreLib/src/System/OperatingSystem.cs

```diff
@@ -103,8 +99,16 @@ public static bool IsOSPlatform(string platform!!)
             return platform.Equals("TVOS", StringComparison.OrdinalIgnoreCase);
 #elif TARGET_ANDROID
```

Comment:
Can we define `const string OSPlatformName = ` using the ifdefs and just use it here to make this less repetitive?

Reply:
done